methods_and_flow_control/string_practice.rb

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+
+string_practice = "this is a string to practice with"
+practice = "practice "
```

Comment:
The purpose of this homework is to help you get familiar with the Ruby String methods.  Can you find a method in https://ruby-doc.org/core-2.2.0/String.html that will result in this substring?  Said another way, you should only need the `string_practice` variable, and some methods called on that variable to produce the output...